blockchain/source/contracts/DevicesStorage.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.4.23;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract DevicesStorage is Ownable {
```

Comment:
why its `Ownable` now? I doesnt see any usage of this.
May be do `kill` function to erase disk space when contract is out of date?